lib/browser/objects-registry.js

```diff
@@ -20,6 +20,7 @@ class ObjectsRegistry {
   add(webContents, obj) {
     // Get or assign an ID to the object.
     let id = this.saveToStorage(obj);
+    this.storage[id].count++;
```

Comment:
Ideally, `remove` is called once for each garbage-collect of the remote reference. Therefore, we need to store the amount of references, so that on the last garbage-collect of the reference, we delete it from `objects-registry`

If we maintain the reference count somewhere else, then sure, we can make sure that we don't remember the amount of references here. Either way, we need to __trace the number of references somewhere__.